configure.ac

```diff
@@ -217,42 +199,89 @@ fi
 AC_SUBST([ENABLED_ASM])
 
 
-# FIPS
+# FIPS 140
 AC_ARG_ENABLE([fips],
     [AS_HELP_STRING([--enable-fips],[Enable FIPS 140-2, Will NOT work w/o FIPS license (default: disabled)])],
     [ENABLED_FIPS=$enableval],
     [ENABLED_FIPS="no"])
 
+if test "$ENABLED_FIPS" != "no"
+then
+    REPRODUCIBLE_BUILD_DEFAULT=yes
+fi
+
+# The FIPS options are:
+#   v5-RC8 - FIPS 140-3 (wolfCrypt WCv5.0-RC8)
+#   v5 - FIPS 140-3 (wolfCrypt v5.0.0)
```

Comment:
It's not actually linux-kernel-specific at all.  it's the only rebased 140-3 codebase in flight, and it has a ton of bug/security fixes in it from the last year that are nothing in particular to do with linuxkm.

i.e. the branch name is a historical accident because my proximal motivation for the branch, long ago, was that.

Reply:
btw this has been updated to
```
# The FIPS options are:
#   v5-RC8 - FIPS 140-3 (wolfCrypt WCv5.0-RC8)
#   v5 - alias for v5-RC8 (may change)
#   v5-REL - FIPS 140-3, placeholder for upcoming wolfCrypt v5.0.0
#   v5-ready - FIPS ready, 140-3 settings
```